### PR TITLE
updated the description on the homepage

### LINKED ISSUE #98: Update HOTH description
image
Current information is out of date.

### PR DESCRIPTION
**Pull request by eric8yang**
Screen Shot 2021-01-14 at 8 19 28 PM
Fixes #98 

changed most of the information, but still need to make "off" in the title bolded (need to import font weight)

### CODE REVIEW

File: src/components/HomePage/HothDescription.js [hunk 1]
```diff
@@ -84,10 +84,11 @@ export default function HothDescription() {
 					</Grid>
 					<Grid item sm={10} md={6}>
 						<Typography variant='body1' style={{ paddingBottom: isSmall ? theme.spacing(2) : 0 }}>
-							Hack on the Hill (HOTH) 7 is a 12-hour,{' '}
+							Hack off the Hill (HOTH 8) is a {' '}
 							<strong>beginner-friendly</strong>
-							{' '}hackathon designed to give beginners a glimpse into what a real
-							hackathon would be and feel like.
+							{' '}hackathon designed to be your first hackathon experience.
+							{' '}Due to the ongoing global pandemic,
```
Comment (TimothyGu) [inline src/components/HomePage/HothDescription.js:90]:
```suggestion
							Due to the ongoing global pandemic,
```

the explicit space is only needed around a JSX tag.

File: src/components/HomePage/HothDescription.js [hunk 2]
```diff
@@ -97,12 +98,11 @@ export default function HothDescription() {
...
-							{` `}<strong>workshops</strong>,
-							{` `}<strong>mentors</strong>, and
-							{` `}<strong>amazing prizes</strong>{' '}
-							for the best hacks. We will also be providing a selection of
-							hardware for hackers to check out and hack with.
+							During HOTH, you{'\''}ll get to learn from
+							{` `}<strong>workshops</strong>, receive technical help from
+							{` `}<strong>mentors</strong>, and meet new people while participating in
+							{` `}<strong>fun</strong> social activities.
+							{` `} There will be <strong>prizes</strong> for the best hacks!
```
Comment (TimothyGu) [inline src/components/HomePage/HothDescription.js:105]: Ditto
```suggestion
							There will be <strong>prizes</strong> for the best hacks!
```

File: src/components/HomePage/HothDescription.js [hunk 3]
```diff
@@ -84,10 +84,11 @@ export default function HothDescription() {
 					</Grid>
 					<Grid item sm={10} md={6}>
 						<Typography variant='body1' style={{ paddingBottom: isSmall ? theme.spacing(2) : 0 }}>
-							Hack on the Hill (HOTH) 7 is a 12-hour,{' '}
+							Hack off the Hill (HOTH 8) is a {' '}
```
Comment (TimothyGu) [inline src/components/HomePage/HothDescription.js:87]:
```suggestion
							Hack off the Hill (HOTH 8) is a{' '}
```

File: src/components/HomePage/HothDescription.js [hunk 4]
```diff
@@ -74,7 +74,7 @@ export default function HothDescription() {
 					<Typography variant='h2' className={classes.subtitle}>What’s</Typography>
 					<Typography variant='h2' className={classNames(classes.title, uppercase.root)} style={{
 						paddingBottom: isSmall ? theme.spacing(2) : theme.spacing(8)
-					}}>Hack on the Hill?</Typography>
+					}}>Hack <Box fontStyle='italic' display='inline'>off</Box> the Hill?</Typography>
```
Comment (TimothyGu) [inline src/components/HomePage/HothDescription.js:77]:
```suggestion
					}}>Hack <em>off</em> the Hill?</Typography>
```

**Review by TimothyGu — COMMENTED** [review]
Do you know how to fix the font? Happy to either give some pointers or let you dig through the code yourself.

**Comment by eric8yang** [comment]
Jody said that she's going to make another issue to fix the fonts, as some are outdated with respect to the style guide

**Comment by TimothyGu** [comment]
The new style guide still uses Poppins, so fixing it would be the right move here. But up to you.

**Review by TimothyGu — APPROVED** [review]

**Review by TimothyGu — APPROVED** [review]

**Pull request merged**